### PR TITLE
Link to other projects using mdbook-i18n-helpers

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

**Review by avivace — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,7 +12,12 @@ Support for translations is a
 
 They've been used successfully to translate [Comprehensive Rust 🦀].
 
-[Comprehensive Rust 🦀]: https://google.github.io/comprehensive-rust/running-the-course/translations.html
+## Use Cases
+
+Please add your project below if it uses this infrastructure for translations:
+
+- [Comprehensive Rust 🦀](https://google.github.io/comprehensive-rust/)
+- [GB ASM Tutorial](https://gbdev.io/gb-asm-tutorial/)
```
Comment (avivace) [inline README.md:20]:
```suggestion
- [Game Boy ASM Tutorial](https://gbdev.io/gb-asm-tutorial/)
```

**Review by avivace — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (avivace) [reply README.md:20]: or: Game Boy Assembly Tutorial

**Review by djmitche — APPROVED** [review]
approved with "Game Boy Assembly Tutorial"

File: README.md (hunk 1 above)
Comment (mgeisler) [reply README.md:20]: To be honest, I kept thinking this was something related to "Great Britain" :smile:  So I like your suggestion a lot:

```suggestion
- [Game Boy Assembly Tutorial](https://gbdev.io/gb-asm-tutorial/)
```

**Pull request merged**